frontend/src/App.js

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import './App.css';
 import { HashRouter as Router, Route, Redirect, Switch,MemoryRouter } from 'react-router-dom';
 import AdminLogin from './Pages/AdminLogin';
-import Dashboard from './Pages/Dashboard';
+import Dashboard from './Pages/Dashboardui';
```

Comment:
It might be better if you don't do the ui separately and the logic separately in the component. For example, you may have a large component in your hand, and if you divide it into parts, you will apply the logic of the component.